### ISSUE TITLE
Cannot get reasonable image as the paper shown

### ISSUE THREAD

**Issue by Godlikegu** [nzilberstein/Repulsive-score-distillation-RSD-#1]
I use the same hyperparameter settings in the runs/half_face.sh and runs/solve_inverse_stable_single.sh, but cannot get reasonable inpainting results

![Image](https://github.com/user-attachments/assets/4144f77a-c9cd-4ae6-9755-0c3124bf7425)

**Comment by nzilberstein** [comment]
Hi, thanks for trying the work, and for raising this. 

You are running the experiments with n_particles = 1, right? If so, can you try with lr_x=0.1 and lr_z=0.2 (divide the ones in the .sh by 4)?

The reason for this is that in the code, I compute the mean of all the losses. Given that I tuned the parameters for 4 particles, for 1 particle the learning rate needs to be modified accordingly (re-scale to have the same learning rate independently of the number of particles). 

I get this with this modification

![Image](https://github.com/user-attachments/assets/51710d30-52ee-4bd2-875f-56c2bd04fea8)


Let me know if it works and anything else.

**Comment by Godlikegu** [comment]
Thank you so much for your good response!

And the main reason is that I use the stable-diffusion-2-1 rather than stable-diffusion-2-1**-base**. I thought this would not affect the results too much. However, it is vital to choose the prior model.

Thank you again for your reply.
